griptape/drivers/vector/local_vector_store_driver.py

```diff
@@ -113,6 +111,18 @@ def query(
                 for r in result
             ]
 
+    def query(
+        self,
+        query: str,
+        *,
+        count: Optional[int] = None,
+        namespace: Optional[str] = None,
+        include_vectors: bool = False,
+        **kwargs,
+    ) -> list[BaseVectorStoreDriver.Entry]:
+        vector = self.embedding_driver.embed_string(query)
+        return self.query_vector(vector, count=count, namespace=namespace, include_vectors=include_vectors, **kwargs)
+
```

Comment:
Can we move this implementation up to the base class since it appears to be common across most implementations?